src/display/api.js

```diff
@@ -178,6 +178,9 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  *   The default value is `false`.
  *   NOTE: It is also necessary to disable streaming, see above,
  *         in order for disabling of pre-fetching to work correctly.
```

Comment:
It looks like the commit message of this commit is copy-pasted from the first one ;-)

Reply:
I've tried to trim the commit messages a bit.